apm-agent-plugins/apm-aws-sdk/apm-aws-sdk-2-plugin/src/main/java/co/elastic/apm/agent/awssdk/v2/BaseSyncClientHandlerInstrumentation.java

```diff
@@ -47,6 +47,11 @@
 import static net.bytebuddy.matcher.ElementMatchers.takesArgument;
 
 public class BaseSyncClientHandlerInstrumentation extends TracerAwareInstrumentation {
+    //Coretto causes sigsegv crashes when you try to access a throwable if it thinks
+    //it went out of scope, which it seems to for the instrumented throwable access
+    private static final boolean IS_CORRETTO = System.getProperty("java.vendor.version") == null ?
+        false :
+        System.getProperty("java.vendor.version").contains("Corretto");
```

Comment:
If eventually we end up doing such check, it should go into `co.elastic.apm.agent.common.JvmRuntimeInfo`